node/src/hopper/routing_service.rs

```diff
@@ -641,23 +626,30 @@ mod tests {
 
         subject.route(inbound_client_data);
 
-        TestLogHandler::new().exists_log_matching(
-            "Couldn't expire CORES package with \\d+-byte payload using main key: DeserializationError.*",
+        TestLogHandler::new().exists_log_containing(
+            "ERROR: RoutingService: Couldn't expire CORES package with 35-byte payload to ProxyClient using main key",
         );
     }
 
+    lazy_static! {
+        static ref REAL_MAIN_CRYPTDE: Box<dyn CryptDE> =
+            Box::new(CryptDEReal::new(TEST_DEFAULT_CHAIN));
+        static ref REAL_ALIAS_CRYPTDE: Box<dyn CryptDE> =
+            Box::new(CryptDEReal::new(TEST_DEFAULT_CHAIN));
+    }
+
     #[test]
     fn logs_and_ignores_message_that_cannot_be_decrypted() {
         init_test_logging();
-        let main_cryptde = main_cryptde();
-        let alias_cryptde = alias_cryptde();
-        let rogue_cryptde = CryptDENull::new(TEST_DEFAULT_CHAIN);
-        let route = route_from_proxy_client(&main_cryptde.public_key(), main_cryptde);
```

Comment:
I'm convinced that you can use the proposed code:

```
 #[test]
    fn logs_and_ignores_message_that_cannot_be_decrypted() {
        init_test_logging();
        let (main_cryptde, alias_cryptde) = {
            //initialization to real CryptDEs 
            let pair = Bootstrapper::pub_initialize_cryptdes_for_testing(&None, &None);
            (pair.main, pair.alias)
        };
        let rogue_cryptde = CryptDEReal::new(TEST_DEFAULT_CHAIN);
        .
        .
```

...and you can delete the lazy_static! implementation up from the test because we can do it with already existing utils. Moreover these `static` variables are referred from this only test. Let's pull them out.